src/frontend/context/SecurityContext.tsx

```diff
@@ -69,6 +71,14 @@ const SecurityProviderInner = ({
     string | null
   >(null);
 
+  const setToUnauthenticated = React.useCallback(() => {
+    if (passcode === null) {
+      throw new Error("Cannot set to unauthenticated without a password set");
+    }
+
+    setAuthState("unauthenticated");
+  }, [passcode]);
+
```

Comment:
used by the listener to set to `unauthenticated`, which renders the passcode screen